src/client/dfs/cont.c

```diff
@@ -1256,11 +1256,21 @@ dfs_get_size_by_oid(dfs_t *dfs, daos_obj_id_t oid, daos_size_t chunk_size, daos_
 	return daos_der2errno(rc);
 }
 
+static bool
+is_uid_invalid(uid_t uid)
+{
+	return uid == (uid_t)-1;
+}
+
+static bool
+is_gid_invalid(gid_t gid)
+{
+	return gid == (gid_t)-1;
+}
+
```

Comment:
just curious why have a function call and not do the == in the if condition below directly?

Reply:
I thought it communicated more clearly what -1 indicated. Figured the compiler will end up inlining this anyway during optimization.

Reply:
I just made a change to explicitly inline it.